apps/server/src/modules/auth/next-auth-options.ts

```diff
@@ -217,10 +219,17 @@ export const NextAuthOptionsProvider: FactoryProvider<NextAuthOptions> = {
             session.user.id = user.id;
             // @ts-expect-error Third part library type mismatch
             session.user.image = user.image ?? user.avatarUrl;
+            // @ts-expect-error Third part library type mismatch
+            session.user.hasPassword = Boolean(user.password);
           } else {
             // technically the sub should be the same as id
             // @ts-expect-error Third part library type mismatch
             session.user.id = token.sub;
+            const databaseUser = await prisma.user.findUnique({
```

Comment:
duplicate database query here

Reply:
`user` in this function params should be the same with the `prisma.user.findUnique`